.changeset/delegation-hook-error-strategy.md

```
@@ -0,0 +1,9 @@
+---
+'@mastra/core': minor
+---
+
+Added `hookErrorStrategy` to `DelegationConfig`, an opt-in strategy for how errors thrown by the `onDelegationStart` and `onDelegationComplete` delegation lifecycle hooks are handled.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)